### ISSUE TITLE
gcsupload completes successfully but artifacts are missing

### ISSUE THREAD

**Issue by kargakis** [openshift/aos-cd-jobs#1173]
```
+ docker run -e 'JOB_SPEC={"type":"presubmit","job":"test_pull_request_openshift_ansible_extended_conformance_install","buildid":"5773","refs":{"org":"openshift","repo":"openshift-ansible","base_ref":"release-3.7","base_sha":"db4cdcdda21e82a4047e7d56f1ce45ef386e5f60","pulls":[{"number":7328,"author":"eriknelson","sha":"3b9e7437b3b9ce477558f3059d70f5e6d35b0fc1"}]}}' -v /data:/data:z registry.svc.ci.openshift.org/ci/gcsupload:latest --dry-run=false --gcs-bucket=origin-ci-test --gcs-credentials-file=/data/credentials.json --path-strategy=single --default-org=openshift --default-repo=origin /data/gcs/artifacts /data/gcs/build-log.txt /data/gcs/finished.json /data/gcs/started.json
Unable to find image 'registry.svc.ci.openshift.org/ci/gcsupload:latest' locally
Trying to pull repository registry.svc.ci.openshift.org/ci/gcsupload ... 
latest: Pulling from registry.svc.ci.openshift.org/ci/gcsupload
6d987f6f4279: Pulling fs layer
186fc760c1fb: Pulling fs layer
d67b2110135d: Pulling fs layer
186fc760c1fb: Verifying Checksum
186fc760c1fb: Download complete
6d987f6f4279: Verifying Checksum
6d987f6f4279: Download complete
d67b2110135d: Verifying Checksum
d67b2110135d: Download complete
6d987f6f4279: Pull complete
186fc760c1fb: Pull complete
d67b2110135d: Pull complete
Digest: sha256:60bd25456c422622c58394c54084b4afd6fa3b0102eb3f81c9629884eda21629
{"component":"gcsupload","level":"info","msg":"Gathering artifacts from artifact directory: /data/gcs/artifacts","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/avc_denials.log in artifact directory. Uploading as artifacts/generated/avc_denials.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/containers.log in artifact directory. Uploading as artifacts/generated/containers.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/dmesg.log in artifact directory. Uploading as artifacts/generated/dmesg.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/docker.config in artifact directory. Uploading as artifacts/generated/docker.config\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/docker.info in artifact directory. Uploading as artifacts/generated/docker.info\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/etcd.conf in artifact directory. Uploading as artifacts/generated/etcd.conf\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/filesystem.info in artifact directory. Uploading as artifacts/generated/filesystem.info\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/installed_packages.log in artifact directory. Uploading as artifacts/generated/installed_packages.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/master-metrics.log in artifact directory. Uploading as artifacts/generated/master-metrics.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/node-metrics.log in artifact directory. Uploading as artifacts/generated/node-metrics.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/generated/pid1.journal in artifact directory. Uploading as artifacts/generated/pid1.journal\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/dnsmasq.service in artifact directory. Uploading as artifacts/journals/dnsmasq.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/docker.service in artifact directory. Uploading as artifacts/journals/docker.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/etcd.service in artifact directory. Uploading as artifacts/journals/etcd.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/openvswitch.service in artifact directory. Uploading as artifacts/journals/openvswitch.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/origin-master-api.service in artifact directory. Uploading as artifacts/journals/origin-master-api.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/origin-master-controllers.service in artifact directory. Uploading as artifacts/journals/origin-master-controllers.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/origin-master.service in artifact directory. Uploading as artifacts/journals/origin-master.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/origin-node.service in artifact directory. Uploading as artifacts/journals/origin-node.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/ovs-vswitchd.service in artifact directory. Uploading as artifacts/journals/ovs-vswitchd.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/ovsdb-server.service in artifact directory. Uploading as artifacts/journals/ovsdb-server.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/journals/systemd-journald.service in artifact directory. Uploading as artifacts/journals/systemd-journald.service\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/scripts/ansible_junit/CokoRoVnKZ.xml in artifact directory. Uploading as artifacts/scripts/ansible_junit/CokoRoVnKZ.xml\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/scripts/ansible_junit/FoXlQJAfpT.xml in artifact directory. Uploading as artifacts/scripts/ansible_junit/FoXlQJAfpT.xml\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/scripts/ansible_junit/zfxpsXhPCk.xml in artifact directory. Uploading as artifacts/scripts/ansible_junit/zfxpsXhPCk.xml\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/scripts/build-images/logs/scripts.log in artifact directory. Uploading as artifacts/scripts/build-images/logs/scripts.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","level":"info","msg":"Found /data/gcs/artifacts/scripts/env/logs/scripts.log in artifact directory. Uploading as artifacts/scripts/env/logs/scripts.log\n","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/scripts/build-images/logs/scripts.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/started.json","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/directory/test_pull_request_openshift_ansible_extended_conformance_install/5773.txt","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/dmesg.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/dnsmasq.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/origin-node.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/scripts/ansible_junit/CokoRoVnKZ.xml","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/filesystem.info","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/origin-master-api.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/ovsdb-server.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/finished.json","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/docker.config","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/docker.info","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/pid1.journal","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/origin-master.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/openvswitch.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/origin-master-controllers.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/systemd-journald.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/build-log.txt","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/scripts/ansible_junit/zfxpsXhPCk.xml","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/containers.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/master-metrics.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/node-metrics.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/scripts/ansible_junit/FoXlQJAfpT.xml","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/avc_denials.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/ovs-vswitchd.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/scripts/env/logs/scripts.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/etcd.conf","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/generated/installed_packages.log","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/docker.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
{"component":"gcsupload","dest":"pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/artifacts/journals/etcd.service","level":"info","msg":"Queued for upload","time":"2018-02-28T18:34:30Z"}
+ exit 0
+ set +o xtrace
########## FINISHED STAGE: SUCCESS: PUSH THE ARTIFACTS AND METADATA [00h 03m 19s] ##########
```
https://ci.openshift.redhat.com/jenkins/job/test_pull_request_openshift_ansible_extended_conformance_install/5773/consoleFull#-43480255458b6e51eb7608a5981914356

https://gcsweb-ci.svc.ci.openshift.org/gcs/origin-ci-test/pr-logs/pull/openshift_openshift-ansible/7328/test_pull_request_openshift_ansible_extended_conformance_install/5773/

@stevekuznetsov unclear yet if this is an issue with gcsupload or something else.

**Comment by stevekuznetsov** [comment]
https://github.com/kubernetes/test-infra/pull/7079 should help us see the end of it ... maybe we somehow lost logs? Also, since we upload in parallel it is very surprising that _everything_ is missing.

**Comment by smarterclayton** [comment]
Are we uploading to the wrong place somehow?

**Comment by stevekuznetsov** [comment]
Not sure it's possible for those logs to have the wrong location string in them, they're using the source of truth to print out

**Comment by kargakis** [comment]
Closing in favor of https://github.com/kubernetes/test-infra/issues/7255
